applications/GeoMechanicsApplication/tests/test_transient_thermal.py

```diff
@@ -154,7 +154,149 @@ def test_unsteady_thermal_heat_flux_3D10N(self):
         temperature = test_helper.get_temperature(simulation)
         temp = temperature[124]
         self.assertAlmostEqual(1.2294110493528096, temp)
+ 
+    def test_thermal_fixed_temperature_2D3N(self):
+        test_name = 'test_thermal_fixed_temperature_2D3N'
+        file_path = test_helper.get_file_path(os.path.join('test_thermal_element', test_name))
+        simulation = test_helper.run_kratos(file_path)
+        temperature = test_helper.get_temperature(simulation)
```

Comment:
Function `test_helper.get_temperature` appears to return a sequence of temperature values (since we index it at the next line). So perhaps we'd better use a plural form to indicate that, e.g. `nodal_temperatures`? This comment also applies to the other new tests in this file.